components/UserPreview/UserPreview.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+import styles from "./UserPreview.module.scss";
+import { Avatar } from "components";
+
+type Props = {
+	name: string;
+	email: string;
+	avatar: string;
+	signupToken: string;
+};
+
+const UserPreview: React.FC<Props> = function ({ name, email, avatar, signupToken }) {
+	return (
+		<a href={`/${signupToken}`} className={styles.previewBlock}>
+			<div className={styles.title}>
+        <Avatar className={styles.avatar} name={name} src={avatar} size={24} />
```

Comment:
Maybe it's a github rendering issue? This `<Avatar>` line was one of the indentation oddities I saw.

Reply:
Oh weird - no, it shows up locally for me in my editor, but prettier doesn't seem to fix it?

Reply:
@isTravis Duh – I ran prettier through ts/js/json/scss, but forgot to include `tsx`. I'll fix that in a sec.
Please see the new PR: https://github.com/underlay/r1/pull/26